tpls/runtime-template.js

```diff
@@ -60,8 +61,13 @@ function install(options) {
                   sendEvent('onUpdating');
                 }
               } break;
-
+              case 'waiting': {
+                sw.skipWaiting();
```

Comment:
Well we could check for skipWaiting if it's undefined, don't do it. I am not familiar enough with the code base here to do it all myself. I think it would be super useful to have this feature. If you want to give me a couple pointers I can attempt it myself.